ambassador/ambassador/config.py

```diff
@@ -995,7 +995,8 @@ def generate_intermediate_config(self):
             if ((module_name == 'ambassador') or
                 (module_name == 'tls') or
                 (module_name == 'authentication') or
-                (module_name == 'tls-from-ambassador-certs')):
+                (module_name == 'tls-from-ambassador-certs') or
+                (module_name == 'cors_default_envoy')):
```

Comment:
Probably this should just be "cors-default" -- but I'm curious why you went with a separate module instead of just adding into the `ambassador` `Module`'s configuration?